src/components/Router/index.js

```diff
@@ -234,13 +234,22 @@ class HistoryRouter extends Component {
     const componentsList = this.componentsList()
     const newStepIndex = currentStep + 1
     if (componentsList.length === newStepIndex) {
-      this.props.options.events.emit('complete')
+      this.triggerOnComplete()
     }
     else {
       this.setStepIndex(newStepIndex)
     }
   }
 
+  triggerOnComplete = () => {
+    const {steps} = this.props.options
```

Comment:
hm, I wonder if we should move this logic to the redux layer, since that's where we do most of our formatting of captures